client/trino-cli/src/main/java/io/trino/cli/QueryRunner.java

```diff
@@ -15,54 +15,37 @@
 
 import io.trino.client.ClientSession;
 import io.trino.client.StatementClient;
+import io.trino.client.uri.HttpClientFactory;
 import io.trino.client.uri.TrinoUri;
 import okhttp3.Call;
 import okhttp3.OkHttpClient;
-import okhttp3.logging.HttpLoggingInterceptor;
 
 import java.io.Closeable;
-import java.sql.SQLException;
 import java.util.concurrent.atomic.AtomicReference;
-import java.util.function.Consumer;
 
+import static com.google.common.base.MoreObjects.firstNonNull;
+import static io.trino.cli.Trino.getCliVersion;
 import static io.trino.client.ClientSession.stripTransactionId;
-import static io.trino.client.OkHttpUtil.setupTimeouts;
 import static io.trino.client.StatementClientFactory.newStatementClient;
 import static java.util.Objects.requireNonNull;
-import static java.util.concurrent.TimeUnit.SECONDS;
 
 public class QueryRunner
         implements Closeable
 {
+    private static final String USER_AGENT = "trino-cli/" + firstNonNull(getCliVersion(), "unknown");
+
     private final AtomicReference<ClientSession> session;
     private final boolean debug;
     private final OkHttpClient httpClient;
-    private final Consumer<OkHttpClient.Builder> sslSetup;
 
     public QueryRunner(
             TrinoUri uri,
             ClientSession session,
-            boolean debug,
-            HttpLoggingInterceptor.Level networkLogging)
+            boolean debug)
     {
         this.session = new AtomicReference<>(requireNonNull(session, "session is null"));
         this.debug = debug;
-
-        OkHttpClient.Builder builder = new OkHttpClient.Builder();
-        try {
-            sslSetup = uri.getSetupSsl();
-            uri.setupClient(builder);
-        }
-        catch (SQLException e) {
-            throw new IllegalArgumentException(e);
-        }
-
-        setupTimeouts(builder, 30, SECONDS);
```

Comment:
Do we want to keep a 30-second timeout as the default for the CLI? It defaults to 10 seconds in OkHTTP.

Reply:
Yes. I don't want to change defaults yet.